runtime/bootstrap/index.ts

```diff
@@ -0,0 +1,141 @@
+import type {Context} from 'isolated-vm';
+import type {FetchRequest} from '../../api_types';
+import type {FetchResponse} from '../../api_types';
+import type {FormulaSpecification} from '../types';
+import type {Isolate} from 'isolated-vm';
+import type {PackFormulaResult} from '../../api_types';
+import type {ParamDefs} from '../../api_types';
+import type {ParamValues} from '../../api_types';
+import type {SyncFormulaResult} from '../../api';
+
+/**
+ * Setup an isolate context with sufficient globals needed to execute a pack.
+ *
+ * Notes:
+ * 1. JSON.parse/stringify are built into v8, so we don't need to inject those.
+ * 2. It is critically important that we do not leak isolated-vm object instances (e.g., Reference, ExternalCopy,
```

Comment:
Probably worth getting thoughts from @chris-codaio too, we obviously aren't doing security-through-obscurity but maybe we want to give some thought to the nature of what we describe here as we start bringing pieces of the runtime into the open-source repo.

Reply:
ok let me give Chris a heads up.

Reply:
Ack

Reply:
haha, I was thinking about tomorrow but apparently you are already anywhere. :P